ships/0027-image-abstraction-layer.md

```diff
@@ -0,0 +1,233 @@
+---
+title: shipwright-image-abstraction-layer
+authors:
+  - "@ricardomaraschini"
+reviewers:
+  - "@ImJasonH"
+  - "@adambkaplan"
+  - "@GabeMontero"
+approvers:
+  - "@ImJasonH"
+  - "@adambkaplan"
+  - "@GabeMontero"
+creation-date: 2022-02-07
+last-updated: 2022-02-07
+status: implementable
+see-also:
+  - "/ships/0020-shipwright-image-api.md"  
+---
+# Shipwright Image Abstraction layer
+
+This enhancement proposal suggests an image management abstraction layer for a Kubernetes cluster
+in the form of an Operator; this abstraction layer, in some aspects, resembles OpenShift’s
```

Comment:
nit: I think we can say it's directly inspired on OpenShift's ImageStream project.

Reply:
Good point. I have adjusted it.